models/recordsModel.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from "mongoose";
+const recordSchema = new mongoose.Schema(
+  {
+    questionId: { type: String, required: true, unique: false },
+    answers: [{ type: Number, requred: true }]
+  },
+  { timestamps: true }
+);
+
+const RecordModel = mongoose.model("Record", recordSchema);
+
+const save = async model => new RecordModel(model).save();
+const getAllRecords = async () => RecordModel.find();
+const getRecordById = async _id => RecordModel.findById({ _id });
+
+const updateById = async (recordId, update) =>
+  RecordModel.findByIdAndUpdate(recordId, update, { new: true });
+
+const deleteRecordById = async recordId =>
+  RecordModel.findByIdAndRemove(recordId);
+
+const getRecordsFromIdArray = async recordId =>
+  RefordModel.find({
+    recordID: { $in: [mongoose.Types.ObjectId(recordId.map())] }
```

Comment:
This won't work. 

- You need to supply key that you will be making query against like _id.
- You are using map function but you do not know how it works. At the end of the day you won't supply array to $in it will be only one object with function inside of it. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map